app/views/schools/participants/index.html.erb

```diff
@@ -5,50 +5,91 @@
+        <div class="group-card">
+          <%= govuk_summary_list(actions: false, borders: nil) do |list| %>
+            <% list.row do |row| %>
+              <% row.key(text: "Mentor") %>
+            <% end %>
+
+            <% list.row do |row| %>
+              <% row.key(text: govuk_link_to(mentor.participant_full_name,
+                                             schools_participant_path(id: mentor.participant_profile_id,
+                                                                      school_id: @school),
+                                             no_visited_state: true),
+                         classes: ["govuk-!-font-weight-regular"]) %>
+              <% row.value(text: render(ParticipantStatusTagComponent.new(profile: mentor.participant_profile,
+                                                                          induction_record: mentor,
+                                                                          has_mentees: true))) %>
+            <% end %>
 
-        <% if participants_active?(@categories.mentor_categories) %>
-          <% component.tab(label: 'Mentors') do %>
-            <%= render partial: 'mentors', locals: { mentor_profiles: @categories.mentor_categories } %>
+            <% list.row do |row| %>
+              <% row.key(text: "ECTs") %>
+            <% end %>
+
+            <% @participants.mentors[mentor].sort_by(&:participant_full_name).each do |ect| %>
+              <% list.row do |row| %>
+                <% row.key(text: govuk_link_to("#{ect.participant_full_name}",
+                                               schools_participant_path(id: ect.participant_profile_id,
+                                                                        school_id: @school),
+                                               no_visited_state: true),
+                           classes: ["govuk-!-font-weight-regular", "govuk-\!-padding-bottom-static-0"]) %>
+                <% row.value(text: render(ParticipantStatusTagComponent.new(profile: ect.participant_profile,
+                                                                            induction_record: ect)),
+                             classes: ["govuk-\!-padding-bottom-static-0"]) %>
+              <% end %>
+            <% end %>
           <% end %>
-        <% end %>
+        </div>
+        <br>
+      <% end %>
 
-        <% if @categories.transferring_in.any? || @categories.transferring_out.any? %>
-          <% component.tab(label: 'Moving school') do %>
-            <%= render partial: 'moving_school', locals: { transferring_in: @categories.transferring_in,
-                                                            transferring_out: @categories.transferring_out } %>
+      <% if @participants.orphan_mentors.any? %>
+        <%= govuk_summary_list(actions: false, borders: nil) do |list| %>
+          <% list.row do |row| %>
+            <% row.key(text: @participants.orphan_mentors.many? ? "Mentors" : "Mentor") %>
```

Comment:
And here. I think there should probably be some (many visually hidden) text here giving a clue as to why they're listed separately. "Mentors without ECTs" perhaps?

Reply:
Done